src/components/Firestore/Collection.tsx

```diff
@@ -123,7 +118,7 @@ export const Collection: React.FC<Props> = ({ collection }) => {
           const doc = state.documents.find(d => d.ref.id === match.params.id);
 
           if (!doc) {
-            return <p>Loading document...</p>;
+            return <></>;
```

Comment:
I kinda feel it's time to get some proper spinners instead of we putting "loading" words and testing for these words...

Reply:
Ack, I'll create a feature-request for loading/error handling - will save for follow ups though.